src/components.d.ts

```diff
@@ -195,6 +213,12 @@ declare global {
     new (): HTMLBdsCardColorElement;
   };
 
+  interface HTMLBdsCheckboxElement extends Components.BdsCheckbox, HTMLStencilElement {}
+  var HTMLBdsCheckboxElement: {
```

Comment:
Prefer `let` or `const`. Add some eslint rule to avoid usage of `var` too 😃

Reply:
file components.d.ts is created automatically on build per stenciljs

Reply:
Sorry 😢